packages/ember-application/lib/system/application.js

```diff
@@ -1097,4 +1031,36 @@ function resolverFor(namespace) {
   return resolve;
 }
 
+function registerLibraries() {
+  if (!librariesRegistered) {
+    librariesRegistered = true;
+
+    if (environment.hasDOM) {
+      Ember.libraries.registerCoreLibrary('jQuery', jQuery().jquery);
+    }
+  }
+}
+
+function logLibraryVersions() {
```

Comment:
I am happy to do whatever here, we just extracted these as-is to make the `init` method easier to read.